.devcontainer/libraries/Dockerfile

```
@@ -0,0 +1,35 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.192.0/containers/dotnet/.devcontainer/base.Dockerfile
+# For details on dotnet specific container, see: https://github.com/microsoft/vscode-dev-containers/tree/main/containers/dotnet
+
+# [Choice] .NET version: 6.0, 3.1
+ARG VARIANT="6.0-focal"
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
```

```suggestion
FROM mcr.microsoft.com/devcontainers/dotnet:0-${VARIANT}
```

As we're starting to migrate contents of [microsoft/vscode-dev-containers](https://github.com/microsoft/vscode-dev-containers) repo to the [devcontainers org](https://github.com/devcontainers), as part of the work on the [open dev container specification](https://containers.dev).

We'll now be publishing images from [devcontainers/images](https://github.com/devcontainers/images) with path `mcr.microsoft.com/devcontainers/*`. Images with path `mcr.microsoft.com/vscode/devcontainers/*` will still be published from the new repo for backwards compatibility, but it will be nice if you could adapt to it now. 😄 

For more details, you can review the [announcement issue](https://github.com/microsoft/vscode-dev-containers/issues/1589)

---

I missed this announcement. Thanks for the review. I'll get this changed.